website/docs/r/cloud_provider_snapshot_restore_job.html.markdown

```diff
@@ -123,6 +123,9 @@ In addition to all arguments above, the following attributes are exported:
 * `target_group_id` -	Name of the target Atlas project of the restore job. Only visible if deliveryType is automated.
 * `target_cluster_name` -	Name of the target Atlas cluster to which the restore job restores the snapshot. Only visible if deliveryType is automated.
 * `timestamp` - Timestamp in ISO 8601 date and time format in UTC when the snapshot associated to snapshotId was taken.
+* `oplogTs` - Timestamp in the number of seconds that have elapsed since the UNIX epoch from which to you want to restore this snapshot. This is the first part of an Oplog timestamp.
+* `oplogInc` - Oplog operation number from which to you want to restore this snapshot. This is the second part of an Oplog timestamp.
+* `pointInTimeUTCSeconds` - Timestamp in the number of seconds that have elapsed since the UNIX epoch from which you want to restore this snapshot.
```

Comment:
We need the full doc here for all 3 new parameters. the API doc or we link to that - I don't think this is enough to actually use this.

Reply:
Also, lines 12-14 are missing the new PIT option so we should add or just remove them (removing is probably fine).  But we also need to add the following from the API page - highlighted at the top:
Important
If you specify "deliveryType" : "automated" or "deliveryType" : "pointInTime" in your request body to create an automated restore job, Atlas removes all existing data on the target cluster prior to the restore.

Reply:
Still need the changes requested here @coderGo93

Reply:
You're right, I did not see that, sorry, I'll add it.